src/app/state/ledger/saga.test.ts

```diff
@@ -28,7 +28,7 @@ describe('Ledger Sagas', () => {
         .dispatch(ledgerActions.enumerateAccounts())
         .put(ledgerActions.setStep(LedgerStep.Done))
         .put.actionType(ledgerActions.accountsListed.type)
-        .run(50)
+        .silentRun(50)
```

Comment:
Do we need to pass a timeout to `silentRun`? another option would be to remove that timeout from `run`. maybe with defaults it would not throw a warning

Reply:
Nope, still shows a warning with defaults